wire-ios-request-strategy/Sources/Synchronization/Decoding/EventDecoder.swift

```diff
@@ -198,33 +198,33 @@ extension EventDecoder {
         publicKeys: EARPublicKeys?,
         proteusService: ProteusServiceInterface
     ) async -> [ZMUpdateEvent] {
+
+        func storeLastEventId() async {
+            await syncMOC.perform {
+                if let eventUUID = event.uuid, !event.isTransient {
+                    self.lastEventIDRepository.storeLastEventID(eventUUID)
+                }
+            }
```

Comment:
question: need to weakify self?
minor: 
```suggestion
            if let eventUUID = event.uuid, !event.isTransient {
                await syncMOC.perform {
                    self.lastEventIDRepository.storeLastEventID(eventUUID)
                }
            }
```

Reply:
no need here

Reply:
good suggestion